sdk/appcenter-data/src/main/java/com/microsoft/appcenter/data/LocalDocumentStorage.java

```diff
@@ -161,23 +175,24 @@ private static SQLiteQueryBuilder getPartitionAndDocumentIdQueryBuilder() {
         return builder;
     }
 
-    private <T> long write(String table, DocumentWrapper<T> document, WriteOptions writeOptions, String pendingOperationValue) {
-        if (writeOptions.getDeviceTimeToLive() == TimeToLive.NO_CACHE) {
-            return 0;
+    private static ContentValues getContentValues(
```

Comment:
https://github.com/microsoft/appcenter-sdk-android/pull/1289#discussion_r337283156 should be summarized as a Javadoc method comment here.

Reply:
Also move it next to the other getContentValues.